### PR TITLE
[doc beta] ember-views & ember-glimmer rfc176 & doc updates

### PR DESCRIPTION
**Pull request by Parrryy**
PR for #15723 for following packages:
- `ember-views`
- `ember-glimmer`

Couldn't find a replacement for `Ember.EventDispatcher`, let me know if there is one so I can change it.

### CODE REVIEW

**Comment by toddjordan** [comment]
You're good.  EventDispatcher still gets namespaced with E ber

File: packages/ember-glimmer/lib/helpers/loc.ts [hunk 1]
```diff
@@ -29,7 +29,7 @@ import { String as StringUtils } from 'ember-runtime';
   </div>
   ```
 
-  See [Ember.String.loc](/api/classes/Ember.String.html#method_loc) for how to
+  See [Ember.String.loc](/api/ember/release/classes/String/methods/loc?anchor=loc) for how to
```
Comment (toddjordan) [inline packages/ember-glimmer/lib/helpers/loc.ts]: Change from `Ember.String.loc` to `String.loc`

**Comment by toddjordan** [comment]
@Parrryy thanks!  

Now it just looks like all that's left is to rebase in the latest master to get rid of the conflict, then squash the files and prepend `[doc beta]`.  let me know if you need help

**Comment by Parrryy** [comment]
The first time, rebasing and had lots of issues so made a new branch and PR. It won't happen again, sorry.

**Pull request closed without merging**